config-model/src/main/java/com/yahoo/vespa/model/search/IndexedSearchCluster.java

```diff
@@ -355,6 +349,9 @@ public void getConfig(DispatchConfig.Builder builder) {
             nodeBuilder.fs4port(node.getDispatchPort());
             builder.node(nodeBuilder);
         }
+        if (useAdaptiveDispatch)
```

Comment:
Please consider "{" on one-line if statements.

Reply:
Normally I do, but I prefer following the consensus in the file. And here all where non-braced.